### DISCUSSION TITLE
如何落地集群部署?

### DISCUSSION THREAD

**Discussion by Powego** [apache/flink-cdc#1068] [Q&A]
根据社区教程, 单机mysql到es成功了, 请问哪位朋友有集群部署教学文档, 能否分享一下,谢谢

**Comment by shadon178** [comment]
集群flink吗？这方面的文档很多的，你随便百度一下就有了。

**Reply by zjf88865** [reply]
flink-cdc只支持flink分布式集群部署嘛？有没有什么办法不通过flink去做flink cdc集群？
